library/src/main/java/com/heartlandpaymentsystems/library/terminals/ConnectionConfig.java

```diff
@@ -152,4 +153,12 @@ public void setDeviceId(String deviceId) {
     public Environment getEnvironment() { return environment; }
 
     public void setEnvironment(Environment environment) { this.environment = environment; }
+
+    public boolean getAllowDupes() {
```

Comment:
Can we update the naming for this flag to spell out "duplicates" (e.g. `getAllowDuplicates`) to more closely align with other SDKs?

Reply:
Sure, will update to match.